.github/workflows/bump-api-schema-sha.yml

```diff
@@ -24,7 +24,7 @@ jobs:
           git config user.name "openapi-getsentry-bot"
 
           filepath="src/build/resolveOpenAPI.ts"
-          sha="$(curl -sSL 'https://api.github.com/repos/getsentry/sentry-api-schema/commits/main' | awk 'BEGIN { RS=",|:{\n"; FS="\""; } $2 == "sha" { print $4 }')"
+          sha="$(curl -sSL 'https://api.github.com/repos/getsentry/sentry-api-schema/commits/main' | awk 'BEGIN { RS=",|:{\n"; FS="\""; } $2 == "sha" { print $4 }')" | head -n 1
```

Comment:
yeah, that will be better. i'll put it together and kick off the branch to test again tx!

Reply:
woo, turns out i fixed it already with the filepath update in https://github.com/getsentry/sentry-docs/pull/9085

<img width="1081" alt="SCR-20240208-ilww" src="https://github.com/getsentry/sentry-docs/assets/187460/31722f94-0240-4fdd-9faa-4babd9d200ac">


i'll still swap to jq with this PR because it's less bananas